packages/eslint-config-base/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@madebywild/eslint-config-base",
+  "version": "0.2.3",
+  "description": "wild's ESLint JS project config",
+  "homepage": "https://github.com/madebywild/eslint-config",
+  "author": "WILD <weare@wild.as> (https://wild.as)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/madebywild/eslint-config/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/madebywild/eslint-config.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [
+    "eslint",
+    "eslintconfig",
+    "config",
+    "wild",
+    "madebywild"
+  ],
+  "main": "index.js",
+  "dependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.25.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-flowtype": "^5.7.2",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-prettier": "^3.4.0"
```

Comment:
afaik, these should be listed as `peerDependencies`, like in other popular configs.
https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/package.json#L67-L69